### PR TITLE
[windows] add PDH helpers

### PR DESCRIPTION
**Pull request by VihasMakwana**
PDH (Performance Data Helper) is a high-level API for collecting performance counters. 

They're already used in metricbeat. See [here](https://github.com/elastic/beats/tree/main/metricbeat/helper/windows/pdh).
We also need the PDH helpers in `elastic-system-metrics` for collecting CPU metrics for [this issue](https://github.com/elastic/beats/issues/40926).

This PR introduces them to `elastic-agent-libs` and we will eventually remove them from `beats`. 

Related https://github.com/elastic/beats/issues/40926

### CODE REVIEW

File: helpers/windows/pdh/pdh_query_windows.go [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+	case PdhFmtDouble:
+		_, value, err := PdhGetFormattedCounterValueDouble(counter.handle)
+		if err != nil {
+			counterValue.Err.Error = err
+			if value != nil {
+				counterValue.Err.CStatus = value.CStatus
+			}
+		} else {
+			counterValue.Measurement = value.Value
+			fmt.Println(value.Value)
```
Comment (leehinman) [inline helpers/windows/pdh/pdh_query_windows.go:334]: This is added from the one in metricbeat.  maybe a leftover debug println?

**Review by leehinman — COMMENTED** [review]
It looks like the `metricbeat/helper/windows/pdh` has definitions for `windows_arm.go` and `windows_arm64.go`.  Is there a reason to skip or include these?

File: helpers/windows/pdh/pdh_query_windows.go (hunk 1 above)
Comment (VihasMakwana) [reply helpers/windows/pdh/pdh_query_windows.go:334]: > This is added from the one in metricbeat. maybe a leftover debug println?

i think it's due to my mistake. I added this.
anyway, I'll remove it

**Review by leehinman — APPROVED** [review]
LGTM once you make the linter happy :-)

**Pull request merged**